src/Sparrow/LowMemory/LowMemoryNotification.cs

```diff
@@ -111,6 +121,39 @@ private void RunLowMemoryHandlers(bool isLowMemory, MemoryInfoResult memoryInfo,
             }
         }
 
+#if NET7_0_OR_GREATER
+        private void RequestLohCompactionIfNeeded(MemoryInfoResult memoryInfo, DateTime now)
+        {
+            if (now - _lastLohCompactionRequest > _lohCompactionInterval && GCSettings.LargeObjectHeapCompactionMode != GCLargeObjectHeapCompactionMode.CompactOnce)
```

Comment:
we probably do not want that in Client API?

Reply:
How can I check that?

Reply:
Done in https://github.com/ravendb/ravendb/pull/16456/commits/249356027e3c12c28b8808d80c9048af90722837